ddtrace/contrib/django/conf.py

```diff
@@ -25,6 +25,12 @@
     'AGENT_HOSTNAME': 'localhost',
     'AGENT_PORT': 8126,
     'AUTO_INSTRUMENT': True,
+    'AUTO_INSTRUMENT_TEMPLATE': {
```

Comment:
don't like the name a lot, maybe we should find something better since `TEMPLATE` is a special keyword in the Django world (and in web frameworks in general).

In general, I like the level nesting of our settings, just to avoid the `AUTO_INSTRUMENT_*` boilerplate. Maybe `INSTRUMENTED_COMPONENTS`? something that is shorter but has the same meaning... have to think about it.

Reply:
here we should remove the nested config and just use `INSTRUMENT_*` for each Django component (cache, database and template)